src/popup/ui.js

```diff
@@ -63,3 +78,73 @@ function createButtonsDiv(node, onReloadItems) {
 	buttons.appendChild(archiveButton)
 	return buttons
 }
+
+function loadLabelSelection(article, labels){
+	document.getElementById('content').style = 'display: none;'
+
+	var labelPage = document.getElementById('labelPage')
+	labelPage.style = 'display: flex;'
+
+	var ckBoxContainer = document.getElementById('labelList')
+
+	labels.forEach((item) => {	
+		var li = document.createElement('li');
```

Comment:
I think instead of using `<ul>` and `<li>` for the label list, it should rather be the HTML structure documented on MDN (just a `<fieldset>` with multiple `<div>`, `<input>` and `<label>` elements): https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input/checkbox#handling_multiple_checkboxes